docs/kitematic/create_pr.md

```diff
@@ -0,0 +1,69 @@
+<!--[metadata]>
++++
+title = "Create a pull request (PR)"
+description = "Explains how to create a pull request for change or new feature"
+keywords = ["Kitematic, open source, contribute, contributor, tour, development, contribute, pull request, review, workflow, beginner, squash,  commit"]
+[menu.main]
+parent= "smn_kitematic_contrib"
+weight=5
++++
+<![end-metadata]-->
+
+# Create a pull request (PR)
+Make sure to follow the <a
+href="http://docs.docker.com/project/create-pr/#rebase-your-branch"
+target="_blank"> rebase guidelines </a> prior to making any pull request.
+This will ensure that your commit is clean and allows for an easier and faster
+merge of your code.
+
```

Comment:
I wasn't sure how much duplicate information we wanted. Fine with me, but will probably be trimmed for the workshop